packages/iov-ethereum/types/integrationtests/atomicswap.spec.d.ts

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
When you change the move-types script in `packages/iov-ethereum/package.json` to (copied from @iov/core)
```
"move-types": "shx rm -r ./types/* && shx mv build/types/* ./types && shx rm ./types/*.spec.d.ts && shx rm ./types/**/*.spec.d.ts",
```

this empty file should disappear. Unfortunatly ** does not work  for no subfolder, so there are two versions of that script.